.github/30-build-installation-issue.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Build/Installation Issue
+about: Use this template for build/installation issues
+
+---
+
+<em>tag:build_template</em>
+
+**Please provide the following information**
+
+**System information**
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- TensorBoard version:
+- TensorFlow version (if different from TensorBoard):
+- Python version:
+- Installed using virtualenv? pip? conda?:
+- Bazel version (if compiling from source):
+- GCC/Compiler version (if compiling from source):
```

Comment:
I don't think we need to ask for these last three fields (compiler, cuda, gpu); they're not really relevant to TensorBoard.